src/nanotron/config/config.py

```diff
@@ -180,13 +203,27 @@ class GeneralArgs:
         ignore_sanity_checks: Whether to ignore sanity checks
     """
 
-    project: str
+    project: Optional[str] = None
     run: Optional[str] = None
+    logs_path: Optional[str] = None
+    launch_slurm_config: Optional[str] = None
+    eval_slurm_config: Optional[str] = None
+    eval_slurm_template: Optional[str] = None
+    lighteval_config_path: Optional[str] = None
+    is_s3_available: Optional[bool] = None
```

Comment:
remove? because we set in it line 406 and and 409?

Reply:
i think it's more clear to have it in the config and define it later no? don't know what's the standard practice here